apps/condo/constants/statusIds.js

```diff
@@ -0,0 +1,13 @@
+const OPEN ='6ef3abc4-022f-481b-90fb-8430345ebfc2'
```

Comment:
We don't need to extract the constants. We have a `TicketStatus.type`. And we also want to support different languages.